### PR TITLE
Minor refactor

### PR DESCRIPTION
**Pull request by dibericky**
Refactor GUI and realigned to given specific

### CODE REVIEW

File: example/src/ApiList.jsx [hunk 1]
```diff
@@ -17,9 +17,9 @@ class ApiList extends React.Component {
   }
 
   componentDidMount() {
-    fetch('https://api.apis.guru/v2/list.json')
-      .then(res => res.json())
-      .then(apis => this.setState({ apis: Object.assign({}, this.state.apis, apis) }));
+    // fetch('https://api.apis.guru/v2/list.json')
```
Comment (davidebianchi) [inline example/src/ApiList.jsx]: Why this part of code is commented?

File: package.json [hunk 2]
```diff
@@ -47,8 +47,10 @@
     "webpack-merge": "^4.1.1"
   },
   "dependencies": {
+    "antd": "^3.16.3",
```
Comment (davidebianchi) [inline package.json]: This could be a peerDependency

File: package.json [hunk 3]
```diff
@@ -47,8 +47,10 @@
...
     "babel-polyfill": "^6.26.0",
     "livereactload": "^4.0.0-beta.2",
+    "lodash": "^4.17.11",
```
Comment (davidebianchi) [inline package.json]: We use a large subset of lodash functions? If we only use lodash.get, import only this

File: packages/api-explorer/src/AuthBox.jsx [hunk 4]
```diff
@@ -56,7 +58,8 @@ function AuthBox({
     <div className={classNames('hub-auth-dropdown', 'simple-dropdown', { open })}>
       {
         // eslint-disable-next-line jsx-a11y/anchor-has-content, jsx-a11y/href-no-hash
-        <a href="#" className="icon icon-user-lock" onClick={toggle} />
+        <Icon type="lock" onClick={toggle} />
+        // <a href="#" className="icon icon-user-lock" onClick={toggle} />
```
Comment (davidebianchi) [inline packages/api-explorer/src/AuthBox.jsx]: Remove old icon

File: packages/api-explorer/src/PathUrl.jsx [hunk 5]
```diff
@@ -16,6 +20,57 @@ function splitPath(path) {
       return { type: part.match(/[{}]/) ? 'variable' : 'text', value: part.replace(/[{}]/g, '') };
     });
 }
+
+function renderButtonTry(loading, onSubmit){
+  /* <button
```
Comment (davidebianchi) [inline packages/api-explorer/src/PathUrl.jsx]: Small tip: 
If you want refactor later, you can set this as //TODO:. If you want fix, // FIXME. If a comment is to remove old code, delete it. 😄

File: packages/api-explorer/src/PathUrl.jsx [hunk 6]
```diff
@@ -30,10 +85,23 @@ function PathUrl({
   oauth,
   auth,
 }) {
+  const containerStyle = {
+    background: '#f0f2f4',
```
Comment (davidebianchi) [inline packages/api-explorer/src/PathUrl.jsx]: We could set colors as js variables

Comment (davidebianchi) [reply packages/api-explorer/src/PathUrl.jsx]: Already used in api-explorer colors

File: packages/api-explorer/src/index.jsx [hunk 7]
```diff
@@ -1,3 +1,10 @@
+import 'antd/dist/antd.css';
```
Comment (davidebianchi) [inline packages/api-explorer/src/index.jsx]: Do not import css, antd should be a peer dep

File: packages/api-explorer/src/index.jsx [hunk 8]
```diff
@@ -6,12 +13,17 @@ const VariablesContext = require('@readme/variable/contexts/Variables');
...
+const {get} = require('lodash')
+const markdown = require('@readme/markdown');
 
 const ErrorBoundary = require('./ErrorBoundary');
 const Doc = require('./Doc');
 
 const getAuth = require('./lib/get-auth');
 
+function getDescription(oasFiles){
+  return get(oasFiles, 'api-setting.info.description')
```
Comment (davidebianchi) [inline packages/api-explorer/src/index.jsx]: description could be undefined?

File: packages/api-explorer/src/lib/oas-to-har.js [hunk 9]
```diff
@@ -94,7 +90,7 @@ module.exports = (
 
   // TODO look to move this to Oas class as well
   if (oas[extensions.PROXY_ENABLED] && opts.proxyUrl) {
-    har.url = `https://try.readme.io/${har.url}`;
+    // har.url = `https://try.readme.io/${har.url}`; // Remove?
```
Comment (davidebianchi) [inline packages/api-explorer/src/lib/oas-to-har.js]: Remove?

File: packages/api-explorer/src/index.jsx (hunk 8 above)
Comment (dibericky) [reply packages/api-explorer/src/index.jsx]: Yes, not all schema have description

**Pull request merged**